report-summary-merged-prs.py

```diff
@@ -1900,6 +1907,13 @@ class PossibleUnitTestResults(object):
                         MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER3,
                         find_one_profiling_result,
                     )
+                    find_general_test_results(
+                        "vtune",
+                        release_queue_results["comparisons"],
+                        arch,
+                        MAGIC_COMMAND_FIND_VTUNE_CHECKS_FILTER,
+                        find_one_vtune_result,
```

Comment:
@gartung , I do not see `find_one_vtune_result` function?

Reply:
added